app/controllers/api/v1/items_controller.rb

```diff
@@ -8,11 +8,21 @@ def show
   end
 
   def create
+    render json: ItemSerializer.new(Item.create(item_params))
   end
 
   def update
+    render json: ItemSerializer.new(Item.update(params[:id], item_params))
   end
 
   def destroy
+    deleted_item = Item.find(params[:id])
```

Comment:
I wonder if there is a better way to do this in which the JSON response will still be returned.